Agent/AgentWar3/UnitRefList/Ability/AbilityInterfaced/CAbilityHero.h

```diff
@@ -16,31 +16,31 @@ class CAbilityHero : CAbilityInterfaced {
     int field_80;
     int field_84;
     int field_88;
-    int experience;
-    int skillPoints;
-    int strength; // assigned with the SetHeroStr native
-    int field_98;
-    int field_9C;
-    int field_A0;
-    int field_A4;
-    int agility; // assigned with the SetHeroAgi native
+    int heroExperience;
+    int heroLevel;
+    int heroStartingStrength;
+    float heroPrimaryAttributeValue;
+    float heroExtraHealth;
+    int heroStartingAgility;
+    float heroArmor;
+    int field_A8;
     int field_AC;
     int field_B0;
     int field_B4;
     int field_B8;
-    int intelligence; // needs to be confirmed
```

Comment:
Can you confirm this?

Reply:
Yes, offset "BC" points to a pointer
![image](https://user-images.githubusercontent.com/13804087/30341349-02be5fb8-97f6-11e7-9904-4e2d6ccd989e.png)